src/libraries/System.Net.Mail/tests/Unit/AssemblyInfo.cs

```diff
@@ -2,5 +2,3 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I think we can remove this file and remove the reference in `src/libraries/System.Net.Mail/tests/Unit/System.Net.Mail.Unit.Tests.csproj`.

Reply:
Addressed in recent commits